From: Dmitry Shachnev <mitya57@debian.org>
Date: Mon, 7 Aug 2017 18:34:10 +0300
Subject: Move sphinxcontrib-websupport to extras_require

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7c1e03d..004df47 100644
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,6 @@ install_requires = [
     'requests>=2.0.0',
     'setuptools',
     'packaging',
-    'sphinxcontrib-websupport',
 ]
 
 extras_require = {
@@ -38,6 +37,7 @@ extras_require = {
         'typing'
     ],
     'websupport': [
+        'sphinxcontrib-websupport',
         'sqlalchemy>=0.9',
         'whoosh>=2.0',
     ],
